app/src/common/shared/com/igalia/wolvic/utils/EnvironmentUtils.java

```diff
@@ -215,23 +215,19 @@ public static Environment getExternalEnvironmentById(@NonNull Context context, @
     }
 
     /**
-     * Returns the URL to the environment's payload, with the SRGB suffix for the devices requiring this
-     * compressed texture format.
+     * Returns the URL to the environment's payload, with the '_alt' suffix for the devices requiring an
+     * alternative compressed texture format (only JPG and PNG are allowed as alternative to KTX).
      * @param env An Environment data structure
      * @return The appropriated URL to the environment's payload .
      */
     @Nullable
     public static String getEnvironmentPayload(Environment env) {
+        // PicoXR doesn't support 'ktx', hence we require that any remote environment must provide a ZIP with an alternative compression format.
+        if (!DeviceType.isPicoXR())
+            return env.getPayload();
         String payload = env.getPayload();
-        String colorSpace = "_rgb", format = "_ktx"; // default configuration
-        if (DeviceType.isOculusBuild())
-            colorSpace = "_srgb";
-        else if (DeviceType.isPicoXR())
-            format = "_jpg";
-        else
-            return payload;
         int at = payload.lastIndexOf(".");
-        return payload.substring(0, at) + format + colorSpace + payload.substring(at);
+        return payload.substring(0, at) + "_alt" + payload.substring(at);
```

Comment:
We should reorganize this. We should have the Pico code inside the if as it's the exception and just do `return payload` as the usual case. So something like
```
String payload = env.getPayload()
if (isPico) {
....
return ;
}
return payload;
```

Reply:
This comment applies to the previous approach, where we provided both rgb and sRGB versions in a single zip. 

As we have discussed, in order to optimize the download operation, we have decided to provide a color space per zip, so now we can't simply return the payload in all the cases by Pico, since not all the devices support sRGB. 

I'll apply your suggestion somehow, though.